server/db/models/products/album.js

```diff
@@ -0,0 +1,59 @@
+const { Model, DataTypes } = require('sequelize');
+const db = require('../../db');
+
+class Album extends Model {}
+
+Album.init({
+  id: {
+    primaryKey: true,
+    type: DataTypes.UUID,
+    defaultValue: DataTypes.UUIDV4
+  },
+  title: {
+    type: DataTypes.STRING,
+    allowNull: false,
+    validate: {
+      notEmpty: true
+    }
+  },
+  description: {
+    type: DataTypes.TEXT
+  },
+  genre: {
+    type: DataTypes.ENUM(['HEAVY METAL','LIGHT JAZZ', 'GLAM ROCK', 'GOSPEL', 'ELEVATOR', 'DISCO', 'HIP-HOP', 'HAIR BANDS', 'MATH ROCK']),
+    allowNull: false
+  },
+  year: {
+    type: DataTypes.INTEGER,
+    allowNull: false,
+    validate: {
+      min: 1700,
+      max: 2021,
+      notEmpty: true
+    }
+  },
+  price: {
+    type: DataTypes.DECIMAL,
```

Comment:
I think price type should look like: DataTypes.DECIMAL(10, 2)
The number 10 is the max integer length (so it could be smaller) and 2 defines 2 decimal places.